src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/move/MovePersister.kt

```diff
@@ -21,12 +22,13 @@ class MovePersister(
 
   fun persist(moves: List<Move>) {
 
-    var counter = 1
+    var counter = 0
     val movesToSave = mutableListOf<Move>()
     val journeysToSave = mutableListOf<Journey>()
     val eventsToSave = mutableListOf<Event>()
     logger.info("Persisting ${moves.size} moves")
 
+    @Transactional
```

Comment:
Moved the transaction boundary to be at the single move level.  This was needed as we don't want to batch insert moves.  The overhead is slower as more network trips.  The big plus is robustness.  It will only fail on the specific move and not the whole import process.